protocol/directory.go

```diff
@@ -7,6 +7,12 @@ import (
 	"github.com/coniks-sys/coniks-go/merkletree"
 )
 
+type ConiksUserPolicies struct {
```

Comment:
Same as https://github.com/coniks-sys/coniks-go/pull/29/commits/5584edca68630937362c755ce085b2a0d156fd81#r72910690 goes for this.